src/legacy/core_plugins/kibana/public/discover/doc_table/components/pager/tool_bar_pager_buttons.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+
+interface Props {
+  hasPreviousPage: boolean;
+  hasNextPage: boolean;
+  onPageNext: () => void;
+  onPagePrevious: () => void;
+}
+
+export function ToolBarPagerButtons(props: Props) {
+  return (
+    <div className="kuiButtonGroup">
```

Comment:
I'm not sure if this is possible, but have you considered using the EuiPagination component instead? https://elastic.github.io/eui/#/navigation/pagination

It looks like it will substitute well from a UI-perspective, which would reduce the dependency upon the deprecated KUI library.

![image](https://user-images.githubusercontent.com/1238659/61226484-5a057f80-a6d7-11e9-80e7-4c7f47336d39.png)

Reply:
I would consider this in a future PR, my current target is to remove angular of discover, I'm aware there is more technical debt here, but I would remove it step by step

Reply:
I will definitly use the `EuiPagination` after saying goodbye to angular.  thank you for the hint